### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,17 @@
...
+  }
+
+  if (days < 7) {
+    return days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
 }
```
Comment (Bogdan-Mykhailov) [inline src/calculateRentalCost.js:44]:
```
 if (days < 3) {
    return days * 40;
  } else if (days < 7) {
    return days * 40 - 20;
  }
  return days * 40 - 50;
```

**Review by Bogdan-Mykhailov — APPROVED** [review]
Good job, this solution is easy to understand, and you can make shorter)

File: src/calculateRentalCost.js (hunk 1 above)
Comment (Andr1yK) [reply src/calculateRentalCost.js:44]: @Bogdan-Mykhailov why do we need to use `else if` instead of just `if`?

**Review by Andr1yK — CHANGES_REQUESTED** [review]
What is 20, 50?
Avoid magic numbers

File: src/calculateRentalCost.js (hunk 1 above)
Comment (Bogdan-Mykhailov) [reply src/calculateRentalCost.js:44]: @Andr1yK if-else statements stop doing comparisons as soon as it finds one that's true.
if-if does every comparison.

Comment (Andr1yK) [reply src/calculateRentalCost.js:48]: > if-if does every comparison

Yes, but not after return :)

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const normalPrice = 40;
+  const discountAfter3days = 20;
+  const discountAfter7Days = 50;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:34]: point 5 - https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -29,6 +29,21 @@
...
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * normalPrice - discountAfter3days;
+  }
+
+  if (days >= 7) {
+    return days * normalPrice - discountAfter7Days;
+  }
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:46]:
```suggestion
  return days * normalPrice - discountAfter7Days;
```

File: src/calculateRentalCost.js [hunk 4]
```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:31]: do u need this comment?

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js [hunk 5]
```diff
@@ -28,7 +28,19 @@
...
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * normalPrice - smallDiscount;
+  }
+
+  return days * normalPrice - bigDiscount;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:43]: [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Save same parts to const

File: src/calculateRentalCost.js [hunk 6]
```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const normalPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * 40;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:36]: hmm.. mb 40 = normalPrice
```suggestion
    return days * normalPrice;
```

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**